core/server/config/index.js

```diff
@@ -179,8 +192,14 @@ ConfigManager.prototype.set = function (config) {
             availableApps:    this._config.paths.availableApps || {},
             clientAssets:     path.join(corePath, '/built/assets/')
         },
+        scheduling: {
+            postScheduling: {
```

Comment:
example case:
We offer the scheduler as a service. but you need to pay for it. because it also provides fancy statistics.
You decide letting post-scheduling run with our service and newsletter locally, because otherwise its too expensive.